README.md

```diff
@@ -65,32 +114,23 @@ const notJsonFiles = FileHound
 ##### Returns
 Returns a FileHound instance.
 
-### `.any(FileHound...)  -> Promise`
-
-##### Parameters
-* Accepts one or more instance of FileHound. Will unpack an array.
-
-##### Returns
-Returns a Promise of all matches. If the Promise fulfills, the fulfullment value is an array of all matching files.
-
-### `.findFiles(path, globPattern) -> Promise`
```

Comment:
Jury is out on this one - Could be convenient for simple searches. However, this could easily be added later if we really find it useful.